changelog.d/574.added.md

```diff
@@ -1 +1 @@
-Support for targetless execution: when not specifying any target for the agent, mirrord now spins up an independent agent. This can be useful e.g. if you are just interested in getting the cluster's dns resolution and outgoing connectivity but don't want any pod's incoming traffic or FS. 
+Support for targetless execution: when not specifying any target for the agent, mirrord now spins up an independent agent. This can be useful e.g. if you are just interested in getting the cluster's DNS resolution and outgoing connectivity but don't want any pod's incoming traffic or FS.
```

Comment:
Wrong changelog?

Reply:
As this was broken and fixed without being released in between, it should not be in the changelog. However I think towncrier requires a changelog change, so I just made a change to an existing entry.

Reply:
Oh, ok.